WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/WebView.swift

```diff
@@ -24,8 +24,24 @@ struct WebView: UIViewRepresentable {
     ///
     var exitTrigger: (() -> Void)?
 
+    /// Callback that will be triggered in onCommit
```

Comment:
We should clarify this further:

```suggestion
    /// Callback that will be triggered in when the underlying `WKWebView` delegate method `didCommit` is triggered.
    /// This happens when the web view has received data and is starting to render the content.
```

Reply:
💯